upf/parallel.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import upf
+from upf.solver import Solver
+from typing import Dict, List, Tuple
+from multiprocessing import Process, Queue
+
+
+class Parallel(Solver):
+    def __init__(self, solvers: List[Tuple[type, Dict[str, str]]]):
+        self.solvers = solvers
+
+    def is_oneshot_planner(self) -> bool:
+        return True
```

Comment:
```suggestion
        return all(Solver(name=s).is_oneshot_planner() for s in self.solvers)
```

This suggests that maybe is_xxx() could be a static method?